docs/jsonapi-spec.md

```diff
@@ -591,6 +628,102 @@ See [Multi-Document Failure Considerations](#multi-document-failure-consideratio
 
 If an error occurs the command will not return `data`.
 
+
+### findCollections Command
+
+`findCollections` returns all valid collection tables from a given namespace.
```

Comment:
"valid" meaning tables that were created as collections, right?

Reply:
Yes but perhaps "valid" is implied and I can remove that qualifier?

Reply:
Ok. Yes, if "valid" is not defined here it probably should be assumed implied. Plus we may need to revisit it anyway, if and when only "JSON-API compatible" tables are exposed; we do not know the details of that yet (afaik, being discussed).